src/Controller/SiteController.php

```diff
@@ -24,4 +31,72 @@ public function index(ServerRequestInterface $request): ResponseInterface
             ]
         );
     }
+
+    public function about(ServerRequestInterface $request): ResponseInterface
+    {
+        return $this->render(
+            'about',
+            [
+                'csrf' => $request->getAttribute(Csrf::REQUEST_NAME)
+            ]
+        );
+    }
+
+    public function contact(
+        ApplicationParameters $app,
+        ContactForm $form,
+        Flash $flash,
+        Mailer $mailer,
+        UrlGeneratorInterface $url,
+        ServerRequestInterface $request
+    ): ResponseInterface {
+        $body = $request->getParsedBody();
+        $method = $request->getMethod();
+
+        if (($method === Method::POST) && $form->load($body) && $form->validate()) {
+            $message = new Message(
+                $form->getAttributeValue('username'),
+                $form->getAttributeValue('email'),
+                $form->getAttributeValue('subject'),
+                $form->getAttributeValue('body'),
+                $app->getEmail(),
+            );
+
+            $attachFiles = $request->getUploadedFiles();
+
+            foreach ($attachFiles as $attachFile) {
+                foreach ($attachFile as $key => $file) {
+                    if ($file->getError() === UPLOAD_ERR_OK) {
+                        $message->addFile($attachFile[$key]);
+                    }
+                }
+            }
+
+            $mailer->send($message);
+
+            $flash->add(
+                'info',
+                [
+                    'header' => 'System mailer notification.',
+                    'body' =>  'Thanks to contact us, we\'ll get in touch with you as soon as possible.'
+                ],
+                true
+            );
+
+            return $this->responseFactory
+                ->createResponse(302)
+                ->withHeader(
+                    'Location',
```

Comment:
How about to use `yiisoft/http` `Header::Location` instead?

Reply:
done.